patches/tModLoader/Terraria/ModLoader/CombinedHooks.cs

```diff
@@ -154,5 +154,58 @@ public static class CombinedHooks
 			}
 			return ret;
 		}
+
+		public static bool CanCaptureWith(Player player, Item item) {
+			bool? canCaptureWithOverall = null;
+			bool? canCaptureWithPlayer = PlayerLoader.CanCaptureWith(player, item);
+			bool? canCaptureWithItem = ItemLoader.CanCaptureWith(item, player);
+			if (canCaptureWithPlayer.HasValue) {
+				if (!canCaptureWithPlayer.Value)
+					return false;
+
+				canCaptureWithOverall = true;
+			}
+			if (canCaptureWithItem.HasValue) {
+				if (!canCaptureWithItem.Value)
+					return false;
+
+				canCaptureWithOverall = true;
+			}
+
+			// I decided to bundle the vanilla logic into this hook, since vanilla does have rules for this
+			// doin' lets me save on patch size and improve readability in-context. it just makes sense
+			// not only that, but if somebody calls this hook elsewhere, that vanilla logic'll be important
+			// -thomas
+			if (canCaptureWithOverall.HasValue)
+				return canCaptureWithOverall.Value;
+			else
+				return item.type == 1991 || item.type == 3183 || item.type == 4821;
+		}
+
+		public static bool? CanCapture(Player player, NPC npc, Item item) {
+			bool? canCaptureOverall = null;
+			bool? canCaptureOnPlayer = PlayerLoader.CanCapture(player, npc, item);
+			bool? canCaptureOnItem = ItemLoader.CanCapture(item, npc, player);
+			bool? canCaptureOnNPC = NPCLoader.CanBeCapturedBy(npc, item, player);
```

Comment:
Should these be called after the early return checks?

Reply:
probably. will do this